tools/istio-iptables/pkg/capture/run.go

```diff
@@ -508,8 +509,8 @@ func (cfg *IptablesConfigurator) Run() error {
 
 	if redirectDNS {
 		// Jump from OUTPUT chain to ISTIOOUTPUT chain for all UDP traffic
-		cfg.ruleBuilder.AppendRule(iptableslog.JumpOutbound, constants.OUTPUT, constants.NAT, "-p", constants.UDP, "-j", constants.ISTIOOUTPUT)
-		cfg.ruleBuilder.AppendRule(iptableslog.JumpOutbound, constants.OUTPUT, constants.RAW, "-p", constants.UDP, "-j", constants.ISTIOOUTPUT)
+		cfg.ruleBuilder.AppendRule(iptableslog.UndefinedCommand, constants.OUTPUT, constants.NAT, "-p", constants.UDP, "-j", constants.ISTIOOUTPUT)
```

Comment:
I think we should replace iptableslog.UndefinedCommand with iptables.JumpOutbound now that the nflog issue is addressed

Reply:
thanks, missed this one